src/coreComponents/physicsSolvers/multiphysics/ThermalSinglePhasePoromechanicsSolver.hpp

```diff
@@ -0,0 +1,104 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file ThermalSinglePhasePoromechanicsSolver.hpp
+ *
+ */
+
+#ifndef GEOSX_PHYSICSSOLVERS_MULTIPHYSICS_THERMALSINGLEPHASEPOROMECHANICSSOLVER_HPP_
+#define GEOSX_PHYSICSSOLVERS_MULTIPHYSICS_THERMALSINGLEPHASEPOROMECHANICSSOLVER_HPP_
+
+#include "codingUtilities/EnumStrings.hpp"
+#include "physicsSolvers/SolverBase.hpp"
+
+namespace geosx
+{
+
+class ThermalSinglePhasePoromechanicsSolver : public SolverBase
+{
+public:
+  ThermalSinglePhasePoromechanicsSolver( const string & name,
+                                         Group * const parent );
```

Comment:
> Regarding the name of the solver, I would put something that specifies clearly that it is a sequential solver based on the fixed stress approach, because I am not sure that the fully coupled thermal-hydro-mechanical solvers will go through the same solver.

Thanks. I've changed the name to `ThermalSinglePhasePoromechanicsFixedStressSolver` and removed `CouplingOption` as this solver is only for fixed stress.